File: CS/WindowsFormsApplication21/CustomDataNavigator.cs
using DevExpress.XtraEditors;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication21 {
    [ToolboxItem(true)]
    public class CustomDataNavigator : DataNavigator {
        public CustomDataNavigator()
            : base() {

        }
        protected override NavigatorButtonsBase CreateButtons() {
            return new CustomDataNavigatorButtons(this);
        }
    }
}

File: VB/WindowsFormsApplication21/CustomDataNavigator.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraEditors
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication21
	<ToolboxItem(True)> _
	Public Class CustomDataNavigator
		Inherits DataNavigator
		Public Sub New()
			MyBase.New()

		End Sub
		Protected Overrides Function CreateButtons() As NavigatorButtonsBase
			Return New CustomDataNavigatorButtons(Me)
		End Function
	End Class
End Namespace
